bin/peggy-cli.js

```diff
@@ -203,9 +213,9 @@ class PeggyCLI extends Command {
         new Option("-O, --optimize <style>")
           .hideHelp()
           .argParser(() => {
-            this.print(this.std.err, "Option --optimize is deprecated from 1.2.0 and has no effect anymore.");
-            this.print(this.std.err, "It will be deleted in 2.0.");
-            this.print(this.std.err, "Parser will be generated in the former \"speed\" mode.");
+            print(this.std.err, "Option --optimize is deprecated from 1.2.0 and has no effect anymore.");
+            print(this.std.err, "It will be deleted in 2.0.");
+            print(this.std.err, "Parser will be generated in the former \"speed\" mode.");
```

Comment:
Because version 3.0 already released, it's time to remove this option

Reply:
moved to #392